org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/SchemaLocation.java

```diff
@@ -43,4 +45,22 @@ public String getLocationHint(String namespaceURI) {
 		return schemaLocationValuePairs.get(namespaceURI);
 	}
 
+	/**
+	 * Given a schema URI, this will return true if the
+	 */
+	public boolean usesSchema(Path rootPath, Path schemaURI) {
+		if (schemaURI == null) {
+			return false;
+		}
+
+		for (String value : schemaLocationValuePairs.values()) {
+			Path currentSchemaURI = Paths.get(value);

```

Comment:
not sure this is a valid use case, but if schemaLocation uses an absolute path it might not be considered in use